code/game/objects/items/mountable_frames/paintings.dm

```diff
@@ -302,7 +349,12 @@ var/global/list/available_paintings = list(
 		if("carp")
 			name = "\improper 'Singing' Mounted Carp"
 			desc = "Too unrobust to beat a carp to death with your bare hands and mount it on a plank of wood? Then this professionally taxidermied trophy is just for you! Note: Does not actually sing."
-
+		if("deer")
+			name = "\improper Mounted Deer head"
```

Comment:
Because the rest were capitalized.

name = "\improper Snow Storm - Steam-Boat off a Harbour's"

name = "\improper Self-portrait"

name = "\improper Cindy Kate"